docs/data-sources/dli_quotas.md

```diff
@@ -0,0 +1,55 @@
+---
+subcategory: "Data Lake Insight (DLI)"
+---
+
+# huaweicloud_dli_quotas
+
+Use this data source to get a list of quotas
+
+## Example Usage
+
+```hcl
+data "huaweicloud_dli_quotas" "test" {}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the resource.
```

Comment:
```suggestion
* `region` - (Optional, String) Specifies the region in which to query the resource quotas.
```

Reply:
This parameter description is automatically generated by PMS.